src/main/kotlin/org/malachite/estella/mails/MailTexts.kt

```diff
@@ -67,11 +68,12 @@ object MailTexts {
             After first login change password and setup your name. We hope that our site will help you with work.
         """.trimIndent()
 
-    fun getTaskAssignmentRequestText(stage: StageType, url: String, hrPartnerFullName: String, position: String): String =
+    fun getTaskAssignmentRequestText(stage: StageType, url: String, hrPartnerFullName: String, position: String, password: UUID?): String =
             """
             Hi,
             You have been chosen as one of people requested to assign a task at stage $stage
             of recruitment process for position $position. You can do that at: $url.
+            Password for this assignment is $password but remember that you can use any of passwords we've sent you that are still valid.
```

Comment:
Ogólnie nie wiem, czy nie lepsze by było jakby hasło było w nowej linii, bo łatwiej by się to kopiowało (UX)

Reply:
Ja się zastanawiałem nad jakimś pogrubieniem, ale pewnie takiej opcji nie mamy, nie? XDDD